lib/index.js

```diff
@@ -58,7 +58,8 @@ var epilogue = {
       pagination: args.pagination,
       updateMethod: this.updateMethod,
       search: args.search,
-      sort: args.sort
+      sort: args.sort,
+      reloadOnCreate: args.reloadOnCreate
```

Comment:
again this relates to my overarching point: a use might not be interested in paying the penalty for that reload. Most people who want to set up an update rest url will not care at all that the data is returned. They will most likely issue a PUT/PATCH, and if they want to see the updated data they will do a GET manually afterwards. What is your use case for *always* needing the updated data after the update operation?

Again, this adds an extra SQL command to every PUT/PATCH